src/components/App.js

```diff
@@ -100,29 +97,32 @@ class App extends React.Component {
       <div className="App">
         <Header />
         <main className="container">
-        <div className="col-1">
 
-        </div>
+          <div className="col-1"></div>
+
           <div className="col-2">
+
             {
               this.state.live
                 ? <Recording live={this.state.live}/>
                 : <h2 className="subTitle">Speak your mind</h2>
             }
-            {this.state.live?<Button style="stop" title="Stop recording" value="stop" handleClick={this.handleStop}/>
-            :<Button style="start" title="Start recording" value="start" handleClick={this.handleStart}/> 
+            {
+              this.state.live
+                ? <Button styles="stop" title="Stop recording" value="stop" handleClick={this.handleStop}/>
```

Comment:
"style" is a keyword, so react was expecting an object instread of a string. Renamed it to "Styles" so as to make proper use of Props